projects/ui/src/shim.cds-core.scss

```diff
@@ -948,11 +948,28 @@
 
   /* Tables */
 
+  // General
+  --clr-table-borderwidth: var(--cds-global-space-1);
+  --clr-table-border-radius: var(--cds-alias-object-border-radius-100);
+  --clr-table-cornercellradius: var(--cds-alias-object-border-radius-100);
+  --clr-table-cell-padding-horizontal: var(--cds-global-space-7);
+  --clr-table-cell-padding-vertical: var(--cds-global-space-5);
+  --clr-table-cell-padding: var(--clr-table-cell-padding-vertical) var(--clr-table-cell-padding-horizontal);
+  --clr-table-cell-compact-padding-vertical: var(--cds-global-space-3);
+  --clr-table-cell-compact-padding: var(--clr-table-cell-compact-padding-vertical) var(--clr-table-cell-padding-horizontal);
+  --clr-table-noborder-borderwidth: var(--cds-global-space-0);
+  --clr-table-cell-padding-left: var(--cds-global-space-7); // ATM delete this property
+  --clr-table-noborder-cell-padding-left: var(--cds-global-space-7); // ATM delete this property
+  --clr-table-zero-padding: var(--cds-global-space-0); // ATM delete this property
```

Comment:
Are we using this somewhere?

Reply:
`--clr-table-zero-padding` looks like remnant. I'll delete it.